### DISCUSSION TITLE
Engaging with higher educational institutes in the UK

### DISCUSSION THREAD

**Discussion by seanmcilroy29** [Green-Software-Foundation/discussions#20] [Ask Anything]
From Chris Lloyd-Jones

I’ve been giving a lot of talks at Tech Skills UK recently - [TechSkills helps develop the digital talent industry needs](https://www.techskills.org/)

Most recently at the University of West London (Skills South), and yesterday, at Manchester Met (Skills North.)

These events represent educators at HEIs, across degrees, apprenticeships, and degree apprenticeships (specifically for the UK.)

Whilst I don’t want to over-weight the foundation in the UK, given one of the roots of our vision is to influence tech education, I was wondering how we could pull the foundation in more.

 TechSkills have asked for a webinar in January – but also how they could onboard university partners, to train their students in green software practices, and add a sustainability module to their curriculum. One of their key asks was PDFs or quick start guides.

**To summarise my ask** – could I get your thoughts & buy-in to engage the community working group to evangelise green software in education? If yes- who is best placed to loop in? If no- is this an activity that could sit elsewhere in the education?

**Comment by danielvaughan** [comment]
How about engaging Google Developer Student Clubs https://developers.google.com/community/gdsc These are established in many universities.

**Reply by jawache** [reply]
Missed this, sounds like a great idea! - cc @Sealjay + @greenhsu123

**Reply by akshitbatra** [reply]
On the similar lines if we are looking for Developers group. 
We can also look for some of GSF member orgs, who already has some education/developer groups. 
1. Microsoft: [Microsoft Learn Student Ambassadors](https://studentambassadors.microsoft.com/)
2. Github: [Github Education](https://education.github.com/)

**Reply by greenhsu123** [reply]
I am unfamiliar with those groups, but could we have a hands-off approach where the Students are driving the engagement with the foundation providing support and guidance? So a couple of ambassadors per university a year to lead this area?

**Reply by akshitbatra** [reply]
@greenhsu123, Generally these groups are community driven and they organize meetups/events, and have different tracks on various topics where technologist come and speak on the subject. Also a great networking opportunities among college students and technologist from the industry.

Collaboration with these groups could be in some of the following ways:
1. Check with the local developer group and see if we can have sustainability/green software as a track in the event.
2. We can collaborate and organize meetup/event together with the developer group targeting students specifically.
3. Identify and check if we can make some of the volunteers from the developer groups /universities as GSF ambassadors, who drive initiatives from their end and connect with GSF for support.

**Comment by akshitbatra** [comment]
How about collaborating with open source/nonprofit developer communities like:

1. [Women who code](https://www.womenwhocode.com/)
> Women Who Code is an international nonprofit organization that provides a global community for women in tech with events, coding resources, jobs, mentorship, and more. They aim to inspire, support, and help women develop technical skills and excel in their careers.

2. [Freecodecamp](https://www.freecodecamp.org/)
> freeCodeCamp is a nonprofit organization that helps people learn to code for free through [thousands of videos](https://youtube.com/freecodecamp), articles, [interactive coding lessons](https://www.freecodecamp.org/learn) and thousands of freeCodeCamp study groups around the world. You learn by completing coding challenges and building projects alongside verified certifications.

**Comment by tmcclell** [comment]
- Asim - next Hackathon - have prizes just for students (several categories, including the above categories). Sponsorship possible.

**Comment by tmcclell** [comment]
@seanmcilroy29 - reach out to educators. @adamj89 - slot to discuss what GSF is doing with academia. Building off exam and book - scalable. Namrata - use Hackathon as an opportunity for healthy competition between universities? Pindy- Opportunities to engage with professors we already have relationships with to discuss gaps. @adamj89 to review.

**Comment by anewmandev** [comment]
This is a great idea - I wasn't aware of this dialogue and have alerady started down this path locally in Brighton so would be intrested to chat and share ideas/progress. I replied on the other thread already... [72](https://github.com/orgs/Green-Software-Foundation/discussions/72)

**Comment by franciscrot** [comment]
At **University of Sussex** I am currently co-leading our School's Green Impact team, and also engaged in a variety of small digital sustainability related projects, including the Digital Humanities Climate Coalition toolkit. Maybe we can chat some time?

**Reply by anewmandev** [reply]
Hi Jo. Yes let's definitely chat. I was on campus today funnily enough talking with a researcher about this stuff. Back again Dec for an event that you should probably be involved in, if not already? I have messaged you on LinkedIn. If better then email me at adam@rootandbranch.io

**Comment by rohan057** [comment]
Hi guys we have a Graduate Programme for 2024 if interested read below and apply.

Graduate Programme 2024: Software Development

About Us:
Accolite Digital, a leading digital transformation services provider, is excited to announce the launch of its Software Development Graduate Programme in 2024.

Position: Software Developer - Python/Java

Location: Glasgow

Programme Start: January 2024

Your Role:
As a Software Developer with Accolite Digital, you will have the opportunity to work on-site, starting in January 2024. Your initial engagement will be with a Fortune 100 global investment bank in the financial services industry. Responsibilities include designing, developing, modifying, and maintaining software products.

What We Offer:

Comprehensive development plan
Extensive and continuous training in banking, financial services, and relevant technologies
Exposure to cutting-edge technology and a dynamic community of technologists
Requirements:

MSc/BSc in a technical-related degree (Computer Science, Computing-Related Fields)
Graduated between 2019-2022
Genuine interest in technology and programming
Coding proficiency in Java and/or Python
Strong understanding of object-oriented concepts
Attention to detail, quality-focused, and adherence to standards/best practices
Ability to thrive in a fast-paced environment
Excellent written and verbal communication skills
About Accolite Digital:
Accolite Digital is an innovative digital transformation services provider, applying design thinking to solve complex business problems for Fortune 500 companies. We believe in a bias-free world and encourage diversity in our workforce.

Why Join Us:
At Accolite Digital, we believe a bias-free world is an empowered world. We welcome applications from everyone, especially women and individuals from diverse backgrounds, to contribute to building the next generation of Accolite Digital.

Ready to Launch Your Career in Software Development?
Apply now for the Software Development Graduate Programme and be part of a community of bright and passionate technologists shaping the future at Accolite Digital.

Python Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37
Java Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37

Reply back if you have any further questions and I will be happy to answer, else best of luck with your applications.
